PMSCore/Permission.cs

```diff
@@ -1,15 +1,31 @@
-﻿namespace PMSCore
+﻿using System.ComponentModel;
+
+namespace PMSCore
 {
     /// <summary>
     /// An enum representing a user's access permission levels.
     /// </summary>
     public enum Permission
     {
-        DEFAULT = 0,
-        USER = 1,
-        EMPLOYEE = 2,
-        MANAGER = 3,
-        ADMIN = 4,
-        SUPERUSER = 5
```

Comment:
do you think we need to enable an option to have multiple permissions by expanding these lines:

    public enum Permission
    {
        DEFAULT = 0,
        USER = 1,
        EMPLOYEE = 2,
        MANAGER = 3,
        ADMIN = 4,
        SUPERUSER = 5

Reply:
No permissions should be on capability bases so different users can share permissions example both an Admin, Programmer and ProjectManager should be able to create, delete and change a task for example

Reply:
Got it, permissions will be based on actions so different roles like Admin and Programmer can share abilities like creating or deleting tasks.